src/dwt.rs

```diff
@@ -0,0 +1,198 @@
+//! Debug and trace and stuff
+
+use cortex_m::peripheral::{DWT,DCB};
+use crate::rcc::Clocks;
+use crate::time::Hertz;
+//use core::u32;
+
+use embedded_hal::blocking::delay::{DelayMs, DelayUs};
+
+/* TODO only on arch7(e)m */
```

Comment:
Not going to be an issue here. STM32F4 will always be armv7em.

Reply:
Ups. I forgot to delete those lines. I ported the from my c-codebase :)